### PR TITLE
feat(IntaveVelocity): improvements

### PR DESCRIPTION
**Pull request by larryngton2**
- ReduceOnAttack: made hurtTime a floatRange value so you can reduce for multiple ticks
- JumpReset: added RandomiseDelay

### CODE REVIEW

**Comment by SuicidalEeper** [comment]
Could you add randomized delay to normal jump reset too while you're at it or is it useless if we can just use this mode?

**Comment by larryngton2** [comment]
> Could you add randomized delay to normal jump reset too while you're at it or is it useless if we can just use this mode?

it is useless, because the normal jumpreset, even on 0 ticks untill jump, doesnt jump on time and actually makes you take more velocity sometimes

**Comment by SuicidalEeper** [comment]
I guess that's a whole new issue that needs fixing then

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/combat/velocity/mode/VelocityIntave.kt [hunk 1]
```diff
@@ -61,13 +63,31 @@ object VelocityIntave : Choice("Intave") {
 
         private val chance by float("Chance", 50f, 0f..100f, "%")
 
+        private inner class RandomiseDelay : ToggleableConfigurable(this, "RandomiseDelay", false) {
```
Comment (1zun4) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/combat/velocity/mode/VelocityIntave.kt]: Randomize

**Comment by 1zun4** [comment]
Please merge upstream.

**Comment by SuicidalEeper** [comment]
Randomize*

**Comment by larryngton2** [comment]
> Randomize*

shit

**Comment by 1zun4** [comment]
PR does not compile.

**Comment by larryngton2** [comment]
> PR does not compile.

i gotta stop commiting on my fucking phone

**Pull request merged**